includes/classes/cache.php

```diff
@@ -93,7 +93,7 @@ function sql_cache_expire_now($zf_query) {
       break;
       case 'database':
       $sql = "delete from " . TABLE_DB_CACHE . " where cache_entry_name = '" . $zp_cache_name . "'";
-      $db->Execute($sql);
+      $db->Execute($sql, false, false);
```

Comment:
if the false, false was not here, what would happen?  it strikes me as odd that for a deletion from the database, one would need to explicitly state to not use the cache as opposed to the framework handling that logic.

Reply:
It's arguably overkill.  If you look at Execute, it disables caching for anything that's not a SELECT.

Reply:
i would say its confusing.  if the vars are going to ignored in the method, why send them?  it confused me.

and why should i need to look at the method?  its just more work for me or any developer/coder/modder (call us what you want).

i love the idea of caching.  i'm just hopeful it does not introduce unwanted errors in all the wrong places.  not everyone uses the zen function to retrieve inventory numbers.  and i support sites that have a number of other inventory buckets, ie in stock, on order, committed, shipped, etc.

we already have this method ```ExecuteNoCache```.  would it not be great if we could simplify?  other classes/functions/scripts  could only call the ```Execute``` or  ```ExecuteNoCache``` with only the sql statement and those functions would then call the class protected function that interacts with the database?   

i know more pain for people upgrading...  perhaps an interim step?

i as thinking about this today, and as people upgrade their ZC sites and their PHP versions, PHP frequently logs warnings about soon to be deprecated methods.  why not start doing that with ZC?  warning, next release of ZC will not accept vars in the Execute method in the db class.  please update....

i know i am asking for too much...  perhaps just some food for thought.  

but if the vars are ignored, please don't send them.  if it's not a ```SELECT``` statement, they are then superfluous, and just confusing to the naked eye.  so don't send them.  MO.